bindings/python/examples/wallet/destroy_foundry.py

```diff
@@ -19,7 +19,7 @@
 foundry_id = "0x08429fe5864378ce70699fc2d22bb144cb86a3c4833d136e3b95c5dadfd6ba0cef0500000000"
 
 # Send transaction.
-transaction = account.destroy_foundry(token_id)
+transaction = account.destroy_foundry(foundry_id)
 
-print(f'Transaction: {transaction.transaction_id}')
-print(f'Block sent: {EXPLORER}/block/" + {transaction.block_id});
+print(f'Transaction: {transaction["transactionId"]}')
+print(f'Block sent: {EXPLORER}/block/" + {transaction["blockId"]}');
```

Comment:
What changed here? We had a proper type and not anymore?

Reply:
No, we never had a type there, this is broken in develop

Reply:
Awesome